jp2_online/templates/estudios_socioeconomicos/estudio_delete_modal.html

```diff
@@ -0,0 +1,20 @@
+<form class="form-horizontal edit_form" method="POST" action="{% url 'captura:estudio_delete'%}">
+  {% csrf_token %}
+  {{ delete_form }}
+  
+  <input type="hidden" name="estudio_id" value="{{ estudio_to_delete.id }}">
+  
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title">Borrar estudio</h4>
+  </div>
+  
+  <div class="modal-body">
+    <p>Esta seguro que desea borrar al estudio con dirección: {{ estudio_to_delete }}</p>
```

Comment:
Fixing the str method of the Family is outside of this PR scope, as it requires rewriting of the api serialiazers. 

The spelling mistake will be fixed.